src/adagrams.js

```diff
@@ -1,15 +1,169 @@
+/**
+ *
+ * creates an array of 10 random letters selected from LETTER_POOL
+ * @return {array of 10 random letters} output
+ */
 export const drawLetters = () => {
-  // Implement this method for wave 1
+  const LETTER_POOL = {
+    A: 9,
+    B: 2,
+    C: 2,
+    D: 4,
+    E: 12,
+    F: 2,
+    G: 3,
+    H: 2,
+    I: 9,
+    J: 1,
+    K: 1,
+    L: 4,
+    M: 2,
+    N: 6,
+    O: 8,
+    P: 2,
+    Q: 1,
+    R: 6,
+    S: 4,
+    T: 6,
+    U: 4,
+    V: 2,
+    W: 2,
+    X: 1,
+    Y: 2,
+    Z: 1,
+  };
+
+  const letterPoolChoices = [];
+  for (let letter in LETTER_POOL) {
+    for (let i = 0; i < LETTER_POOL[letter]; i++) {
+      letterPoolChoices.push(letter);
+    }
+  }
```

Comment:
Nice use of two different types of loops! We can do `const letter` because it is never reassigned within the loop.